### PR TITLE
Document release instructions

### PR DESCRIPTION
**Pull request by naweidner**
Added section on how to release Opencast Studio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,3 +145,25 @@ npm run build
...
+## How to release Opencast Studio:
+
+Checkout master-branch from Opencast Studio (working directory should be clean!). 
+Execute './create-release.sh'. Now there should be two new *.tar.gz files.
+
+### Release:
+
+Go to https://github.com/elan-ev/opencast-studio/releases and click "Draft a new release". Click on "choose a tag" and enter the current date in YYYY-MM-DD and then "Create new tag".
+Make sure target is still 'master'.
+As release title enter the current date again in the same format.
```
Comment (lkiesow) [inline README.md:158]:
```suggestion
As a release title, enter the current date again in the same format.
```

File: README.md [hunk 2]
```diff
@@ -145,3 +145,25 @@ npm run build
...
+List all changes (since the last release) in the text box.
+Upload the two *.tar.gz files and click "Publish release".
+
+### Update Opencast:
```
Comment (lkiesow) [inline README.md:162]:
```suggestion
### Update Opencast
```

File: README.md [hunk 3]
```diff
@@ -145,3 +145,25 @@ npm run build
...
+Checkout a new branch. 
+Go to modules/studio/pom.xml. You have to change two lines:
+  <opencast.studio.url> -> change the url to the new relase-url from *integrated.tar.gz. (Usually only the date has to be adjusted here.)
+  <opencast.studio.sha256> -> update the sha256 hash. (Run sha256sum *-integrated.tar.gz in Opencast Studio folder)
+ Open a new PR (optional: add link to release and changelog in description)
```
Comment (lkiesow) [inline README.md:167]:
```suggestion
 Open a new pull request (optional: add a link to the release and changelog in the description)
```

File: README.md [hunk 4]
```diff
@@ -145,3 +145,25 @@ npm run build
...
+ 
+ Befor opening an new PR, you can build Opencast and test Opencast Studio (The release date should be on the info info page)
```
Comment (lkiesow) [inline README.md:169]:
```suggestion
Before opening a new pull request, you can build Opencast and test Opencast Studio. The release date should be on the info page.
```

**Review by lkiesow — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -145,3 +145,26 @@ npm run build
...
+Go to [the release section on GitHub](https://github.com/elan-ev/opencast-studio/releases) and click "Draft a new release". Click on "choose a tag" and enter the current date in YYYY-MM-DD and then "Create new tag".
+Make sure the target is still `master`.
+As a release title, enter the current date again in the same format.
+List all changes (since the last release) in the text box.
+Upload the two `*.tar.gz` files and click "Publish release".
+
+### Update Opencast
+
+Checkout a new branch. 
+Go to modules/studio/pom.xml. You have to change two lines:
```
Comment (LukasKalbertodt) [inline README.md:165]:
```suggestion
Go to `modules/studio/pom.xml`. You have to change two lines:
```

**Review by LukasKalbertodt — CHANGES_REQUESTED** [review]

**Comment by LukasKalbertodt** [comment]
I was so bold as to resolve all Lars' comments as they were fixed exactly as requested. I also added one suggestion and immediately committed it via GitHub, so this is now mergable!

**Pull request merged**